src/client/components/MainFooter/MainFooter.tsx

```diff
@@ -99,61 +100,45 @@ function getEl(astElement: AstNode | AstNode[], index: number = 0) {
         );
       case 'li':
         return (
-          <li className={styles.Link} key={index}>
-            {children}
-          </li>
+          <UnorderedList.Item className={styles.Link} key={index}>
+            <Icon svg={ChevronRightIcon} size="level-5" />
+            <div>{children}</div>
+          </UnorderedList.Item>
         );
       case 'strong':
-        return <strong>{children} </strong>;
+        return (
+          <>
+            <strong>{children}</strong>&nbsp;
+          </>
+        );
```

Comment:
Volgens mij is {' '} de preferred way in React om een whitespace te renderen.

Reply:
prima